src/_content/sponsors/jbssolutions.md

```diff
@@ -0,0 +1,17 @@
+---
+hidden: false
+level: Gold
+name: JBS Solutions
+date: 2025-08-31
+logo:
+    filename: jbslogo.png
+    orientation: landscape
+url:
+    target: 'https://www.jbs.dev/'
+    label: www.jbs.dev
+hiring_url: 'https://www.jbs.dev/our-team/careers/'
+---
+JBS is a highly experienced team of senior developers and technology trailblazers obsessed with accelerating business
+innovation at any size. From high-growth startups to Fortune 500 organizations, we are problem-solvers orchestrating
+software development with the right harmony of passion and pragmatism to closely align with your business needs and
+growth.
```

Comment:
I meant these extra line breaks. Do you notice how in your screenshot, there are arbitrary line breaks in the copy?